policyengine_canada/parameters/gov/provinces/sk/tax/income/credits/dividend_tax_credit/fraction/other_than_eligible.yaml

```diff
@@ -0,0 +1,12 @@
+description: Saskatchewan multiplies the other than eligible taxable amount of dividends under the total dividend tax credit by this amount.
+values: 
+  2022-01-01: 0.01695
+metadata:
+  unit: /1
+  period: year
+  label: Saskatchewan dividend tax credit other than eligible dividends fraction
+  reference: 
+  - title: 5008-D Worksheet SK428 - Saskatchewan 2022 
+    href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/5008-d/5008-d-22e.pdf#page=3
+  - title: Saskatchewan The Income Tax Act, 2000, Dividend credit, c25, s.13; 2020, c3-29*, s.3; 2022, c46, s.3.
+    href: https://pubsaskdev.blob.core.windows.net/pubsask-prod/806/I2-01.pdf#page=31
```

Comment:
page 31 doesn't have any 1.695%

Reply:
1.695% is in the first reference: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/5008-d/5008-d-22e.pdf#page=3. The second reference is a Canada-wide reference, so it does not contain the parameter of the SK policy.